src/ansys/mechanical/core/embedding/app.py

```diff
@@ -115,8 +115,14 @@ def new(self):
         self.DataModel.Project.New()
 
     def close(self):
-        """Close the application."""
+        """Close the application and remove the lock file."""
```

Comment:
change the docstring to:

"""Close the active project."""

Let's add a new function `exit` for closing the application, in which we can internally call `Application.Close`, I will also research in the backend what we have to do in `exit` in order to properly terminate everything